docs/docs/policies.mdx

```diff
@@ -678,44 +675,22 @@ of the tracker has a couple steps:
        * features indicating the results of any API calls stored in
          slots, e.g. `slot_matches`
 
-       * features indicating what the last action was (e.g.
+       * features indicating what the last bot action or bot utterance was (e.g.
          `prev_action_listen`)
 
-2. **Convert all the features into numeric vectors**:
-
-   We use the `X, y` notation that's common for supervised learning,
-   where `X` is an array of shape
-   `(num_data_points, time_dimension, num_input_features)`,
-   and `y` is an array of shape `(num_data_points, num_bot_features)`
-   or `(num_data_points, time_dimension, num_bot_features)`
-   containing the target class labels encoded as one-hot vectors.
-
-   The target labels correspond to actions taken by the bot.
-   To convert the features into vector format, there are different
-   featurizers available:
-
-     * `BinarySingleStateFeaturizer` creates a binary one-hot encoding:
-
-         The vectors `X, y` indicate a presence of a certain intent,
-         entity, previous action or slot e.g. `[0 0 1 0 0 1 ...]`.
+       * features indicating if any loop is active and which one
 
-     * `LabelTokenizerSingleStateFeaturizer` creates a vector
+2. **Convert all the features into numeric vectors**:
 
-         based on the feature label:
-         All active feature labels (e.g. `prev_action_listen`) are split
-         into tokens and represented as a bag-of-words. For example, actions
-         `utter_explain_details_hotel` and
-         `utter_explain_details_restaurant` will have 3 features in
-         common, and differ by a single feature indicating a domain.
+    `SingleStateFeaturizer` uses rasa nlu to convert the intent and
+    bot action or bot utterance names into numeric vectors.
+    Please see [NLU Model Configuration](./model-configuration.mdx)
+    for the details on how features are represented.
+    Bot action or bot utterance names are featurized by nlu pipeline
+    similarly as intent names.
```

Comment:
similarly? Isn't it the same?

Reply:
by similarly I mean that they use the same pipeline but different vocabs

Reply:
Actually, I think the last sentence can be removed